Formula/n/newnode-helper.rb

```diff
@@ -0,0 +1,39 @@
+class NewnodeHelper < Formula
+  desc "Web proxy that uses a distributed p2p network to circumvent censorship"
+  homepage "https://www.newnode.com/newnode-vpn"
+  url "https://github.com/clostra/newnode.git",
+    tag: "2.1.4", revision: "c42a04ded55cfdff857878f4d9234950e530bb00"
+  license "GPL-2.0-only"
+
+  depends_on xcode: ["9.3", :build]
+  depends_on "automake"
+  depends_on "coreutils"
+  depends_on "libtool"
+  depends_on "mbedtls@2"
+  depends_on "wget"
```

Comment:
Is this needed outside the test?

Reply:
We can't support that hardcoded path.

Reply:
Actually, it's not needed at all, the build script compiles HTTPRequest.m instead of https_wget.c for mac os
https://github.com/clostra/newnode/blob/master/build.sh